frontend/src/config.js

```diff
@@ -11,6 +11,7 @@ const config = {
   KEYCLOAK_REALM: window.REACT_APP_KEYCLOAK_REALM || process.env.REACT_APP_KEYCLOAK_REALM,
   COMS_URL: window.REACT_APP_COMS_URL || process.env.REACT_APP_COMS_URL,
   COMS_BUCKET: window.REACT_APP_COMS_BUCKET || process.env.REACT_APP_COMS_BUCKET,
+  SHOW_EXPERIMENTAL_FEATURES: (window.REACT_APP_SHOW_EXPERIMENTAL_FEATURES || process.env.REACT_APP_SHOW_EXPERIMENTAL_FEATURES) ?? false,
```

Comment:
I've added new environmental variable `REACT_APP_SHOW_EXPERIMENTAL_FEATURES` (set as true/false) and default to false